### PR TITLE
Create "Change Status" Modal for Admins on Admin Dashboard

### PR DESCRIPTION
**Pull request by jbarlow96**

### CODE REVIEW

**Comment by jbarlow96** [comment]
closes #8

File: src/app/dashboard/learning-object-management/learning-object-management.component.ts [hunk 1]
```diff
@@ -43,6 +43,9 @@ export class LearningObjectManagementComponent implements OnInit, OnDestroy {
   total: number;
   restrictions = Object.keys(Restriction);
   focusedObject: LearningObject;
+  adminStatuses = Object.keys(Status);
```
Comment (DonSeannelly) [inline src/app/dashboard/learning-object-management/learning-object-management.component.ts]: I'm not sure that this is the best name. These aren't so much the statuses of an admin, but more so those of a Learning Object.

File: src/app/dashboard/learning-object-management/learning-object-management.component.ts [hunk 2]
```diff
@@ -43,6 +43,9 @@ export class LearningObjectManagementComponent implements OnInit, OnDestroy {
...
+  statusObject: LearningObject;
```
Comment (DonSeannelly) [inline src/app/dashboard/learning-object-management/learning-object-management.component.ts]: I also don't think that this is the clearest name either. Perhaps something like `activeLearningObject`?

File: src/app/dashboard/learning-object-management/learning-object-management.component.ts [hunk 3]
```diff
@@ -43,6 +43,9 @@ export class LearningObjectManagementComponent implements OnInit, OnDestroy {
...
+  statusModal = false;
```
Comment (DonSeannelly) [inline src/app/dashboard/learning-object-management/learning-object-management.component.ts]: If this boolean is just being used to determine when to show the modal on screen, the name should reflect that. Something like `displayStatusModal` would be better.

File: src/app/dashboard/learning-object-management/learning-object-management.component.ts [hunk 4]
```diff
@@ -181,6 +185,20 @@ export class LearningObjectManagementComponent implements OnInit, OnDestroy {
     this.focusedObject = object;
   }
 
+  openStatusModal() {
```
Comment (DonSeannelly) [inline src/app/dashboard/learning-object-management/learning-object-management.component.ts]: Make sure each function you add here gets JSDoc.

File: src/app/dashboard/learning-object-management/learning-object-management.component.ts [hunk 5]
```diff
@@ -244,4 +267,15 @@ export class LearningObjectManagementComponent implements OnInit, OnDestroy {
       (<string[]>this.focusedObject.lock.restrictions).includes(restriction.toString().toLowerCase())
       : false;
   }
+
+  // Returns the current status the selected learning object has previously been given.
```
Comment (DonSeannelly) [inline src/app/dashboard/learning-object-management/learning-object-management.component.ts]: This should be JSDoc instead of just a single-line comment.

**Pull request merged**